diff --git a/frontend/OBSApp.cpp b/frontend/OBSApp.cpp
index 0d6d0e2..f8e46f0 100644
--- a/frontend/OBSApp.cpp
+++ b/frontend/OBSApp.cpp
@@ -950,13 +950,13 @@ OBSApp::OBSApp(int &argc, char **argv, profiler_name_store_t *store)
 		crashHandler_ = std::make_unique<OBS::CrashHandler>(appLaunchUUID_);
 	}
 
-	sleepInhibitor = os_inhibit_sleep_create("OBS Video/audio");
+	sleepInhibitor = os_inhibit_sleep_create("MoonClip video/audio");
 
 #ifndef __APPLE__
-	setWindowIcon(QIcon::fromTheme("obs", QIcon(":/res/images/obs.png")));
+	setWindowIcon(QIcon::fromTheme("moonclip", QIcon(":/res/images/obs.png")));
 #endif
 
-	setDesktopFileName("com.obsproject.Studio");
+	setDesktopFileName("dev.souriscg.moonclip");
 
 	pluginManager_ = std::make_unique<OBS::PluginManager>();
 }
